packages/appstore/src/components/CFDs/cfd-demo-accounts.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { CFD_PLATFORMS } from '@deriv/shared';
+import AccountManager from '../account-manager';
+import { TCFDAccountsProps, TPlatform, TDetailsOfEachMT5Loginid, TStaticAccountProps, TRootStore } from 'Types';
+import { useStores } from 'Stores/index';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+
+const CFDDemoAccounts = ({ isDerivedVisible, isFinancialVisible, current_list }: TCFDAccountsProps) => {
+    const available_demo_accounts: Array<TStaticAccountProps> = [
+        {
+            name: 'Derived',
+            description: localize('Trade CFDs on MT5 with Derived indices that simulate real-world market movements.'),
+            is_visible: isDerivedVisible(CFD_PLATFORMS.MT5),
+            disabled: false,
+            platform: CFD_PLATFORMS.MT5,
+            type: 'synthetic',
+        },
+        {
+            name: 'Financial',
+            description: localize('Trade CFDs on MT5 with forex, stocks & indices, commodities, and cryptocurrencies.'),
+            is_visible: isFinancialVisible(CFD_PLATFORMS.MT5),
+            disabled: false,
+            platform: CFD_PLATFORMS.MT5,
+            type: 'financial',
+        },
+        {
+            name: 'Deriv X',
+            is_visible: isDerivedVisible(CFD_PLATFORMS.DXTRADE),
+            description: localize(
+                'Trade CFDs on Deriv X with Derived indices, forex, stocks & indices, commodities and cryptocurrencies.'
+            ),
+            disabled: false,
+            platform: CFD_PLATFORMS.DXTRADE,
+            type: 'synthetic',
+            // ToDo: deriv x should have type of all in new API
+            // type: 'all'
+        },
+    ];
+
+    const { client, modules, common, ui }: TRootStore = useStores();
+    const {
+        standpoint,
+        dxtrade_tokens,
+        createCFDAccount,
+        setCurrentAccount,
+        setMT5TradeAccount,
+        toggleMT5TradeModal,
+        enableCFDPasswordModal,
+        openAccountNeededModal,
+        has_maltainvest_account,
+    } = modules.cfd;
+    const { platform, setAppstorePlatform } = common;
+    const { openTopUpModal } = ui;
+    const { is_eu } = client;
```

Comment:
```suggestion
    const { platform, setAppstorePlatform } = common;
    const { openTopUpModal } = ui;
    const { is_eu } = client;
```
Does copying these properties out of the store still makes it observable?